workflows-scripts/marc-authority/view-marc-authority/marc_viewAuthorityRecord.jmx

```diff
@@ -0,0 +1,863 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.4.3">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="marc_viewAuthorityRecord" enabled="true">
+      <stringProp name="TestPlan.comments">This test plan was created by EPAM performance team</stringProp>
```

Comment:
Please provide a more practical comment such as a description of the workflows being tested.